docker/build/devdeps.Dockerfile

```diff
@@ -76,7 +76,7 @@ RUN LLVM_INSTALL_PREFIX=/opt/llvm LLVM_SOURCE=/llvm-project \
         source scripts/install_toolchain.sh -e /opt/llvm/bootstrap -t ${toolchain}
 RUN source /opt/llvm/bootstrap/init_command.sh && \
     LLVM_INSTALL_PREFIX=/opt/llvm \
-        bash /scripts/build_llvm.sh -s /llvm-project -c Release -v \
+        bash /scripts/build_llvm.sh -s /llvm-project -c DEBUG -v \
```

Comment:
I suspect we don't want this change. (Debug builds are bigger, slower, and should be used by the developer at his/her own discretion, etc.)